sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableServiceClientTest.java

```diff
@@ -76,6 +76,15 @@ protected void beforeTest() {
         serviceClient = getClientBuilder(false).buildClient();
     }
 
+    @Test
+    public void listTables() {
+        serviceClient.listTables().iterableByPage().forEach(page -> {
+            page.getElements().forEach(tableItem -> {
+                System.out.printf("Table name: %s%n", tableItem.getName());
```

Comment:
This test is not making any assertions. Maybe it was accidentally left in?

Reply:
@vcolin7 This was a simple test I used when having issues connecting to the service. I'll remove it.